rich/console.py

```diff
@@ -522,14 +522,19 @@ def _is_jupyter() -> bool:  # pragma: no cover
         return False  # Other type (?)
 
 
-COLOR_SYSTEMS = {
+ConsoleColorSystem = Literal["auto", "standard", "256", "truecolor", "windows"]
```

Comment:
This name isn't too descriptive. Perhaps `ColorSystemName` ?

It's also not too accurate, given that `auto` is not a color system, it tells the Console to auto detect one of the other values.